pages/roadmap.js

```diff
@@ -0,0 +1,153 @@
+                <div>
+                  <h2 className="font-semibold mb-2 lg:my-0 lg:text-center lg:text-2xl">
+                    <div className="uppercase inline-block bg-gray-100 text-green-400 text-xs p-1.5 mr-2 lg:block lg:bg-transparent lg:text-green-400 lg:text-sm lg:p-0 lg:mr-0 lg:mb-2">Goal</div>
+                    AsyncAPI unifies all the API specifications
+                  </h2>
+                  <p className="my-4 text-gray-600 lg:text-center">
+                    Nobody does only event-driven architectures. Most people complement them with REST (OpenAPI), GraphQL, and/or RPC APIs. <strong>We want people to use the AsyncAPI specification and tooling together with their existing OpenAPI, GraphQL, and gRPC definitions.</strong> For that purpose, our specification and tools will need to understand and leverage many other specifications and tools.
+                  </p>
+                  <p className="my-4 text-gray-600 lg:text-center">
+                    This is not about reinventing the wheel or creating yet another spec to solve the same problems but to integrate with the existing tools instead.
+                  </p>
+                </div>
+                <div>
+                  <h2 className="font-semibold mt-6 mb-2 lg:my-0 lg:text-center lg:text-2xl">
+                    <div className="uppercase inline-block bg-gray-100 text-green-400 text-xs p-1.5 mr-2 lg:block lg:bg-transparent lg:text-green-400 lg:text-sm lg:p-0 lg:mr-0 lg:mb-2">Goal</div>
+                    API development experience is seamless
+                  </h2>
+                  <p className="my-4 text-gray-600 lg:text-center">
+                    We want to make the development experience super seamless, from idea to production. <strong>Users should be able to create their first API within minutes without prior knowledge of AsyncAPI, the communication protocol, or anything else that's not business logic.</strong>
+                  </p>
+                  <p className="my-4 text-gray-600 lg:text-center">
+                    Furthermore, making sure production behavior and documentation are always aligned, independently of the programming language or framework of choice.
+                  </p>
+                </div>
+                <div>
+                  <h2 className="font-semibold mt-6 mb-2 lg:my-0 lg:text-center lg:text-2xl">
+                    <div className="uppercase inline-block bg-gray-100 text-green-400 text-xs p-1.5 mr-2 lg:block lg:bg-transparent lg:text-green-400 lg:text-sm lg:p-0 lg:mr-0 lg:mb-2">Goal</div>
+                    The AsyncAPI community grows 400%
+                  </h2>
+                  <p className="my-4 text-gray-600 lg:text-center">
+                    We are aware that our goals are insane. <strong>None of the other goals are possible if we don't have a huge community supporting us. AsyncAPI is and must continue being a community-driven initiative. Now more than ever.</strong>
```

Comment:
```suggestion
                    We are aware that our goals are ambitious. <strong>None of the other goals are possible if we don't have a huge community supporting us. AsyncAPI is and must continue being a community-driven initiative. Now more than ever.</strong>
```
As per SIG call discussion and @lornajane's suggestion.

Reply:
Fixed! Thanks, Mike and Lorna!